SU2_CFD/include/solvers/CSolver.hpp

```diff
@@ -3799,6 +3799,13 @@ class CSolver {
    */
   inline virtual su2double GetAverageDensity(unsigned short valMarker, unsigned short valSpan) const { return 0.0; }
 
+  /*!
+   * \brief virtual member
+   * \param[in] val_marker - boundary marker
+   * \return Value of the mass flow rate on the surface <i>val_marker</i>
+  */
+  inline virtual su2double GetAverageMassFlowRate(unsigned short valMarker) const {return 0.0; }
```

Comment:
```suggestion
  inline virtual su2double GetTargetMassFlowRate(unsigned short valMarker) const {return 0.0; }
```

Reply:
This isn't the target it is the calculated value, the average indicates that is the sum of all the spanwise values